extensions/datasource/runtime/src/main/java/io/quarkus/datasource/runtime/DataSourceBuildTimeConfig.java

```diff
@@ -16,4 +16,13 @@
     @ConvertWith(DatabaseKindConverter.class)
     public Optional<String> dbKind = Optional.empty();
 
+    /**
+     * Whether this particular data source should be excluded from the health check if
+     * the general health check for data sources is enabled.
+     * <p>
+     * By default, the health check includes all configured data sources (if it is enabled).
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean healthExclude;
```

Comment:
That will make it `health-exclude` not `health.exclude`. Also shouldn't we have a test?

Reply:
Hmm. Maybe we should keep it at `health-exclude`, because`quarkus.datasource.health.enabled` is a setting that is only used globally, whereas the exclusions are per each datasource. If we used `health.enabled` here, then we would get
- `quarkus.datasource.health.enabled` for the global setting
- `quarkus.datasource.health.exclude` would be to exclude the DEFAULT data source, but it would look very similar to the previous one (it shares the complete segment namespace) and therefore it would look like it's a global setting too, but it's not... Hope my explanation is understandable

Anyway I've added a test as you requested

Reply:
Hey @gsmet so what do you think? We got a user asking when this will be released. Still insist on renaming?

Reply:
Hi @gsmet we need this for many production container, can we help on this PR so that can be merged soon?

Thanks